test/data/00/README.md

```diff
@@ -13,5 +14,3 @@ $$f \equiv 0$$
 | MiniZinc |  Spin  |   ✔️    |
 | QUBO     |  Bool  |   ✔️    |
 | Qubist   |  Spin  |   ✔️    |
-
-[symbols]: # (✔️❌)
```

Comment:
I guess since there are no more missing in this table we do not need the legend, but I would just leave it to overemphasize that there are others out there